charts.py

```diff
@@ -0,0 +1,145 @@
+import os
+import json
+import numpy as np
+
+from flask import (
+    Flask, flash, render_template,
+    redirect, request, session, url_for)
+from flask_pymongo import PyMongo
+
+if os.path.exists('env.py'):
+    import env
+
+
+charts = Flask(__name__)
+charts.config['MONGO_DBNAME'] = os.environ.get('MONGO_DBNAME')
+charts.config['MONGO_URI'] = os.environ.get('MONGO_URI')
+charts.secret_key = os.environ.get('SECRET_KEY')
+
+mongo = PyMongo(charts)
+
+# collection
+data_set_coll = mongo.db.SampleData
+
+
+@charts.route('/chart_label_one', methods=['GET'])
+def chart_lable_one():
```

Comment:
I think it should be `label` vs `lable`. Also, I would reverse the naming and use nace instead of label here to get `nace_one_chart`, `nace_two_chart` etc.